lib/email/utils/helpers.js

```diff
@@ -39,13 +70,34 @@ function logFlowEventFromMessage(log, message, type) {
       flow_time: elapsedTime
     }
 
+    // If a locale exists and log it
+    if (locale) {
+      flowEventInfo.locale = locale
+    }
+
     log.flowEvent(flowEventInfo)
   } else {
     log.error({ op: 'handleBounce.flowEvent', templateName, type, flowId, flowBeginTime, currentTime})
   }
 }
 
+function getAnonymizedEmailDomain(email) {
+  // This function returns an email domain if it is considered a popular domain,
+  // which means it is in `./config/popular-email-domains.js`. Otherwise, it
+  // return `other` as domain.
+  const tokens = email.split('@')
+  const emailDomain = tokens[1]
+  var popularEmailDomain = 'other'
```

Comment:
naming nit: this var could now be "anonymizedEmailDomain" for consistency with the name of the function.